document_definition/models/document_definition.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+# © 2017 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import api, fields, models
+
+
+class DocumentDefinition(models.Model):
+    _name = 'document.definition'
+
+    term = fields.Char(help='Term to define', required=True)
+    definition = fields.Text(help='Definition of term', required=True)
+
+    # stored compute field used for making the letter grouping
+    # need to be stored in order to use grouping
+    first_letter = fields.Char()
```

Comment:
If it is to be a stored computed field, any reason why you aren't just using `compute=` and `stored=`?

Reply:
computed fields are not allowed in groupings, as we use it. Only database persistent data is allowed.
Logically I thought also that a stored computed field would had met this requirement, but odoo refuses to accept any computed field as a grouping function.